httpmw/cognito_user.go

```diff
@@ -2,41 +2,41 @@ package httpmw
 
 // CognitoUser cogntito user entity.
 type CognitoUser struct {
-	username string
-	groups   []string
-	lookup   map[string]struct{}
+	Username string
+	Groups   []string
+	Lookup   map[string]struct{}
```

Comment:
I believe these fields were reverted from a previous version, and they should be kept private since we're using a get/set pattern on `CognitoUser`.

Reply:
I needed to make them exported fields to marshal  user instances into []bytes to store in redis.

Reply:
To understand the issue with using private fields for (un)marshaling into bytes, here is a little example:

With private fields : https://go.dev/play/p/Xrv9V69gYnf

With exported fields : https://go.dev/play/p/7o1ZNN4U58t